Server/src/helpers/userValidation.js

```diff
@@ -47,19 +44,13 @@ class UserValidations {
   */
   static validateSignin(req, res, next) {
     try {
-
       req.body.email = req.body.email.trim();
       req.body.password = req.body.password.trim();
-
-      if (!email.test(req.body.email)) throw new Error('please, insert and make sure it\'s valid email');
-      if (!((req.body.password).length > 5)) throw new Error('please, insert and make sure it\'s valid password');
-
+      if (!email.test(req.body.email)) throw new Error('invalid email');
+      if (!password.test((req.body.password))) throw new Error('invalid password');
```

Comment:
Am wondering why you changed the error message from `'please, insert and make sure it\'s valid email'` this is more descriptive?